src/bond/james.cljc

```diff
@@ -33,6 +33,22 @@
                       (vec))
      (do ~@body)))
 
+(defmacro with-spy-namespaces
```

Comment:
Bikeshed:

`clojure.core`'s functions related to namespaces use `ns` rather than `namespace` - `create-ns`, `find-ns`. I would name this `with-spy-ns` to match those.

Or maybe just name it `with-spies` 🤷‍♂️